src/cljs/org/broadinstitute/firecloud_ui/common/input.cljs

```diff
@@ -13,7 +13,7 @@
    :validate
    (fn [{:keys [props state this]}]
      (let [text (react/call :get-text this)
-           fails (keep (fn [p] (when-not ((:test p) text) (:message p))) (:predicates props))]
+           fails (keep (fn [p] (when-not ((:test p) text) (:message p))) (keep identity (:predicates props)))]
```

Comment:
Also, we should probably take out the assert (on line 22), that's no longer a requirement.